Composer/packages/extensions/intellisense/src/components/IntellisenseTextField.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ITextFieldProps, TextField } from 'office-ui-fabric-react/lib/TextField';
+import React from 'react';
+import { useLanguageServer } from '../hooks/useLanguageServer';
+import { checkIsOutside } from '../utils/uiUtils';
+import CompletionList from './CompletionList';
+
+export const IntellisenseTextField = (
+  props: {
+    url: string;
+    scopes: string[];
+    projectId?: string;
+    id: string;
+    value?: string;
+    onChange: (newValue: string) => void;
+  } & Omit<Partial<ITextFieldProps>, 'onChange'>
```

Comment:
This feels strange to omit an expected event and then redefine it with a different prototype.

Reply:
This was to conform to the FieldProps onChange definition (packages/extensions/extension/src/types/form.ts) while enabling some ITextField customization if needed.

Reply:
This goes back to the comment I had on FieldProps<T> and the onChange expecting the param to be of type T.  Other fields expose an onChange with the event which the wrapping component then has handleChange to raise the onChange<T>.  In this case you are exposing the onChange<T> directly so you don't need the handleChange.  This is fine, but there isn't consistency between the intellisense fields and the other fields.